api-error-response.go

```diff
@@ -100,7 +100,10 @@ func httpRespToErrorResponse(resp *http.Response, bucketName, objectName string)
 		msg := "Response is empty. " + reportIssue
 		return ErrInvalidArgument(msg)
 	}
-	var errResp ErrorResponse
+
+	errResp := ErrorResponse{
+		StatusCode: resp.StatusCode,
+	}
```

Comment:
IMO `resp.StatusCode` should be added to the `errResp` var being assigned below as well